api/v2/helpers/bouquets/bouquets_helper.py

```diff
@@ -9,3 +9,12 @@ def query_all_bouquets():
         bouquets = BouquetsSchema(many=True).dump(all_bouquets)
 
     return bouquets
+    
+def query_one_bouquets(bouquet_id):
+    one_bouquet = BouquetsModel.query.filter_by(id=bouquet_id).first()
+    get_bouquet = False
```

Comment:
`get_bouquet` is a very bad variable name. Please resist using these such verbs as variable names!